### ISSUE TITLE
Timeline: DateTime.MONTH is null or not an object

### ISSUE THREAD

**Issue by GoogleCodeExporter** [berusch/simile-widgets-old#366]
```
Runs fine in Firefox, but returns this error in IE

Message: 'Timeline.DateTime.MONTH' is null or not an object
Line: 799
Char: 9
Code: 0
URI: http://www.coloradoan.com/article/99999999/MULTIMEDIA02/399990060



```

Original issue reported on code.google.com by `kganno...@gmail.com` on 16 Apr 2010 at 4:23

**Comment by GoogleCodeExporter** [comment]
```
Strange. It appears to no longer be an issue.
```

Original comment by `kganno...@gmail.com` on 16 Apr 2010 at 5:26

**Comment by GoogleCodeExporter** [comment]
```
We see this issue as well with IE7 and IE8, but only randomly.  FF never pops 
the error.  When we do see it in IE, a simple browser refresh or two will 
eventual get past the error.  This occurs for timeline v2.3.1 and "pre 2.4.0"

We consider this to be critical as our main users are only allowed to use IE.
```

Original comment by `codebene...@gmail.com` on 1 Feb 2011 at 7:20

**Comment by GoogleCodeExporter** [comment]
```
Two things seemed to help as we have not seen the problem in a while. 1) We 
stopped using the local simile bundle in favor of the individual files 
(bundle=false).  2) We noticed in the code that Timeline.DateTime.* is 
deprecated in favor of SimileAjax.DateTime.*, so we use that instead.
```

Original comment by `codebene...@gmail.com` on 23 Mar 2011 at 2:04

**Comment by GoogleCodeExporter** [comment]
Original comment by `ryan...@csail.mit.edu` on 23 Jun 2011 at 9:37
- Added labels: **Project-Timeline**

**Comment by GoogleCodeExporter** [comment]
```
Thanks! 
SimileAjax.DateTime.* worked
```

Original comment by `TheOutla...@gmail.com` on 9 Mar 2013 at 12:42
